ansys/dpf/core/field_base.py

```diff
@@ -403,31 +407,31 @@ def _set_data_pointer(self,data):
         
     @property
     def data(self):
-        """Access the data of this field.    
+        """Data in the field as an array.    
         
         Notes
         -----
-        Print a progress bar
+        Print a progress bar.
         
         Returns
         -------
-        data : numpy.ndarray
-            Data of this field.
+        numpy.ndarray
+            Data in the field.
         """
         return self._get_data()
     
     @property
     def data_as_list(self):
-        """The data of this field in a python list
+        """Data in the field as a Python list.
 
         Returns
         -------
-        data : list
-            Data of this field.    
+        List
+            List of the data in the field.    
         
         Notes
         -----
-        Print a progress bar
+        Print a progress bar.
```

Comment:
Saw this several times.  I'm assuming this is a TODO.  If so, should be a comment in the source not in the docstring.  @cbellot000, is that the case?

Reply:
@akaszynski no it's not a to do, it's done, I didn't know how to mention that something is printing out during the evaluation